### PR TITLE
feat(ui5-dynamic-page): proper docs added

### PR DESCRIPTION
**Pull request by PetyaMarkovaBogdanova**
Dynamic Page docs added.
Dynamic behaviour added to the buttons inside Dynamic page basic story.

### CODE REVIEW

**Comment by nnaydenow** [comment]
Rebase to main and align with latest jsdoc standard. You can use documentation in #8123.

**Comment by PetyaMarkovaBogdanova** [comment]
> Rebase to main and align with latest jsdoc standard. You can use documentation in #8123.

Thanks, I was just intending to do the same

File: packages/playground/_stories/fiori/DynamicPage/DynamicPage.stories.ts [hunk 1]
```diff
@@ -859,6 +858,36 @@ Basic.args = {
...
+    ${story()}
+<script>
+    const page = document.getElementById("page");
+    const scrollDownStep = 500;
+
+    function getScrollContainer() {
+        return page.shadowRoot.querySelector(".ui5-dynamic-page-scroll-container");
+    }
+
+    document.getElementById("toggleFooterBtn").addEventListener("click", () => {
```
Comment (dobrinyonkov) [inline packages/playground/_stories/fiori/DynamicPage/DynamicPage.stories.ts:873]: This script tag throws an error in the console and prevents modifying the example using the storybook controls. If I remember correctly we talk that they are not needed, as they were only added for testing purpose.

File: packages/playground/_stories/fiori/DynamicPage/DynamicPage.stories.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ const Template: UI5StoryArgs<DynamicPage, StoryArgsSlots> = (args) => {
     >
```
Comment (dobrinyonkov) [inline packages/playground/_stories/fiori/DynamicPage/DynamicPage.stories.ts:48]:
```jsx
    ?header-snapped="${ifDefined(args.headerSnapped)}"
    ?header-pinned="${ifDefined(args.headerPinned)}"
    ?show-footer="${ifDefined(args.showFooter)}"
    >
```
This needs to be adjusted as above in order to correctly propagate the values

**Review by dobrinyonkov — APPROVED** [review]

**Pull request merged**